lib/debug/server_cdp.rb

```diff
@@ -1059,8 +1059,8 @@ def process_cdp args
                 result = b.local_variable_get(expr)
               rescue NameError
                 # try to check method
-                if M_RESPOND_TO_P.bind_call(b.receiver, expr, include_all: true)
```

Comment:
Interestingly, `respond_to?`'s `include_all` argument is positional, not keyword (notice `=`, not `:` in the docs):

> `respond_to?(symbol, include_all=false) → true or false`


The way this is called causes an implicit Hash to be passed positionally, as if it was:

```ruby
if M_RESPOND_TO_P.bind_call(b.receiver, expr, { include_all: true })
```

This would cause `include_all`'s value to be literally the Hash `{ include_all: true }`. It's truthy, so it behaves equivalent to `true`, so it happens to be accidentally correct. This wouldn't work though:

```ruby
if M_RESPOND_TO_P.bind_call(b.receiver, expr, include_all: false)
```

Since the `{ include_all: false }` Hash would still be truthy.

I made this a kwarg in the Reflection helper, so it's more clear.